triton-client/src/components/Landing/index.tsx

```diff
@@ -49,9 +49,62 @@ function LandingPage({ isLoggedIn, userName }: UserInformationProps) {
                     />
                 </Tooltip>
             </Content>
-            <Text className="Common-subtitle Landing-user" type="secondary">
-                {isLoggedIn ? `Logged in as ${userName}` : "Logging In.."}
-            </Text>
+            <Content className="landing-page-content">
+                <Text className="Common-subtitle Landing-user" type="secondary">
+                    {isLoggedIn ? `Logged in as ${userName}` : "Logging In.."}
+                </Text>
+                <Typography.Title level={2}>Guide</Typography.Title>
+                <Content className="faq-container">
+                    <div className="faq-section-container">
+                        <Typography.Title level={5}>
+                            1. The Triton app landing page is where you will
```

Comment:
Skip the tour info on the landing page FAQ ... Go straight to the staging stuff (last paragraph stuff). Indicate more specifically the expected states displayed on the buttons. Make sure to indicate that you can only chose to use one method (SFTP or Globus) for each dataset. You can follow up with the password reset stuff. Underline the importance to unstage manually downloaded datasets to allow the staging of other datasets.

Reply:
Eh if Ulysse thinks that, then I guess I can skip that section ;)

Reply:
@UlysseFG  Curious to know, did you actually visit the page, or did you just look at the code?